Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -393,24 +348,24 @@ HRESULT _DWriteCreateFontFamilyWithName(CFStringRef familyName, IDWriteFontFamil
 HRESULT _DWriteCreateFontFaceWithName(CFStringRef name, IDWriteFontFace** outFontFace) {
     // Parse the font name for font weight, stretch, and style
     // Eg: Bold, Condensed, Light, Italic
-    _DWriteFontProperties properties = _DWriteGetFontPropertiesFromName(name);
+    std::shared_ptr<_DWriteFontProperties> properties = _DWriteGetFontPropertiesFromName(name);
 
     // TODO: #1250: Need to be able to load fonts from the app's bundle
     // For now return a default font to avoid crashes in case of missing fonts
     // When #1250 is completed, remove this
-    if (!properties.familyName) {
+    if (!properties->familyName.get()) {
         name = CFSTR("Segoe UI");
```

Comment:
I think I talked about this before?
- this is code that will eventually be removed.
- this is only used here.
- there are default font constants elsewhere, pulling this into a constant would probably add to confusion.

---
In reply to: [87664357](https://github.com/Microsoft/WinObjC/pull/1366#discussion_r87664357) [](ancestors = 87664357)